py4DSTEM/io/datastructure/py4dstem/calibration.py

```diff
@@ -132,7 +132,7 @@ def set_R_pixel_size(self,x):
     def get_R_pixel_size(self):
         return self._get_value('R_pixel_size')
     def set_Q_pixel_units(self,x):
-        pix = ('pixels','A^-1')
+        pix = ('pixels','A^-1','mrad')
```

Comment:
not sure if `'mrad'` is our best option, maybe should consider `'arcmilliseconds'` instead